airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py

```diff
@@ -3,13 +3,35 @@
 #
 
 import logging
-from typing import Any, Dict, Iterable
+from typing import Any, Dict, Iterable, Mapping
 
+import fastavro
 from airbyte_cdk.sources.file_based.config.file_based_stream_config import FileBasedStreamConfig
 from airbyte_cdk.sources.file_based.file_based_stream_reader import AbstractFileBasedStreamReader
 from airbyte_cdk.sources.file_based.file_types.file_type_parser import FileTypeParser
 from airbyte_cdk.sources.file_based.remote_file import RemoteFile
 
+AVRO_TYPE_TO_JSON_TYPE = {
+    "null": "null",
+    "boolean": "boolean",
+    "int": "integer",
+    "long": "integer",
+    "float": "number",
+    "double": "string",  # double -> number conversions can lose precision
```

Comment:
I think this will need to be configurable to be backwards compatible with existing avro sources. 

For parquet, we added a config field that [defaults to converting to strings](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/file_based/config/parquet_format.py#L12), but [default to numbers if the config is legacy](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py#L86-L99).

Reply:
fair point yeah i can add a similar thing to avro as well. Sucks have the extra overhead but backwards compatibility is more important at the moment